src/libValidator/Validator.cpp

```diff
@@ -205,7 +205,8 @@ bool Validator::CheckCreatedTransactionFromLookup(const Transaction& tx)
         return false;
     }
 
-    return AccountStore::GetInstance().UpdateAccountsTemp(
-        m_mediator.m_currentEpochNum, tx);
+    // return AccountStore::GetInstance().UpdateAccountsTemp(
+    //     m_mediator.m_currentEpochNum, tx);
+    return true;
```

Comment:
Reason for such changes? Could have explained a bit in the comment

Reply:
The transaction comes from lookup will also be processed in SubmitTransaction, where all the transaction will do UpdateAccountsTemp, remove this line to avoid it being processed twice.